Contest_API.md

```diff
@@ -770,6 +770,65 @@ The `countdown_pause_time` may be changed to indicate the approximate delay unti
 Countdown is resumed by setting a new `start_time` and resetting
 `countdown_pause_time` to `null`.
 
+In most cases downstream clients should use `/scoreboard` endpoint to get standings. If for some reason, it's not suitable, one
+can use `scoreboard_rules` property to ensure that calculated scoreboard would be the same, or at least report potential
+diverges. Scoreboard rule object must have a `string` property named `kind` and any additional properties, describing settings. 
+CCS should use one of the objects described in the [known scoring rules](#known-scoreboard-rules) section, when possible.  
+
+#### Known scoreboard rules
+
+##### ICPC scoreboard rules
+
+`ICPC` kind is set of scoreboard calculation rules, where each problem is either solved or not, teams
+are sorted based on number of solved problems, and penalty time as a tiebreaker. Can be used only with `pass-fail` 
+`scoreboard_type`.
+
+| Name                                  | Type     | Description
+|:--------------------------------------|:---------| :----
+| kind                                  | string   | Should be eqaul to ICPC
+| penalty\_time\_per\_wrong\_submission | RELTIME  | Penalty time team gets for a wrong submission on a problem solved after that. Defaults to `00:20:00`.
+| penalty\_mode                         | string   | One of values: `sum`, `last`, `ignore`. Defaults to `sum`
+| penalty\_precision                    | RELTIME  | Precision of penatly calculation. Typically 1 minute or 1 second. Defatuls to `00:01:00`.
+| penalty\_rounding                     | string   | One of values: `down`, `up`, `closest`. Defaults to `down`.
+| penalty\_rounding\_point              | string   | One of values: `each_submission`, `total`. Defaults to `each_submission`.
+
+`penalty_mode` describes how times of submissions results in final penalty:
+* `sum` means that penalty time is the sum of submission times
+* `last` means that penalty time is the time of the last submission
+* `ignore` means that times are not used in penalty calculation, only wrong attempts matter.
```

Comment:
Is it really "ICPC" if you use `last` or `ignore`?

I'm not arguing that we should drop this, but rather that this "kind" is larger than just "ICPC" if we include these.